### ISSUE TITLE
RSS Feed for Blog

### ISSUE THREAD

**Issue by jonashrem** [OpenMage/OpenMage.github.io#44]
An RSS Feed on https://www.openmage.org/blog.html would be nice ;)

**Comment by nickw108** [comment]
There is an RSS feed at https://www.openmage.org/feed.xml, although there is an error on line 82 at column 830: `xmlParseEntityRef: no name` which seems to prevent me from subscribing to it in Slack. 

>Sorry, ‘https://www.openmage.org/feed.xml’ doesn’t look like a feed url. Try again?

In Feedly I was able to subscribe to it, though.

**Comment by jonashrem** [comment]
Doesn't work in thunderbird im afraid

![grafik](https://user-images.githubusercontent.com/1112507/121189068-0885b100-c86a-11eb-87cc-0cb4d109112b.png)

See also https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwww.openmage.org%2Ffeed.xml

**Comment by Flyingmana** [comment]
seems to be a bug from https://github.com/jekyll/jekyll-feed which does not properly encode the `&` in category names

**Comment by Flyingmana** [comment]
the patch is breaking a few parts of the page...
but the feed works now without an error.

**Comment by jonashrem** [comment]
yes, thunderbird accepts the feed now as well. Thanks @Flyingmana 

Can I close this issue or is there anything left to do?

**Comment by Flyingmana** [comment]
I would prefer to leave it open, so someone can check this more in depth because I did only change some obious places, 

and now the display in https://www.openmage.org/2019/08/18/new-home-magento-connect-modules.html is not so nice anymore.
and https://www.openmage.org/blog-patches-and-addons does not show anything at all anymore, so I messed something up 😅

**Comment by jonashrem** [comment]
ok

**Comment by sreichel** [comment]
Dont know how "new home ..." content looked before, but patches & categories issue is fixed in https://github.com/OpenMage/OpenMage.github.io/pull/140.
